test/brick_seeker_web/features/searching_for_parts_test.exs

```diff
@@ -20,4 +16,16 @@ defmodule BrickSeeker.SearchingForPartsTest do
     end)
     |> PartsIndexPage.assert_part_selected(part_to_select)
   end
+
```

Comment:
I got rid of the listing behavior of index, since we want to highlight the autocomplete searching for now. These tests can probably be deleted for now.